lib/Backend/Lower.cpp

```diff
@@ -7692,6 +7684,17 @@ Lowerer::LowerUnaryHelperMemWithBoolReference(IR::Instr *instr, IR::JnHelperMeth
     return this->LowerUnaryHelperMem(instr, helperMethod, useBoolForBailout ? boolOpnd : nullptr);
 }
 
+IR::Instr *
+Lowerer::LowerInitCachedScope(IR::Instr* instr)
+{
+    instr->m_opcode = Js::OpCode::CallHelper;
```

Comment:
Yes, sorry I missed that.

---
In reply to: [67801018](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r67801018) [](ancestors = 67801018,67748513)